src/taskpane/excel.js

```diff
@@ -3,12 +3,39 @@
  * See LICENSE in the project root for license information.
  */
 
-/* global document, Office, require */
+/* global document, Excel, Office */
 
-const ssoAuthHelper = require("./../helpers/ssoauthhelper");
+import { getGraphData } from "./../helpers/ssoauthhelper";
+import { filterUserProfileInfo } from "./../helpers/documentHelper";
 
 Office.onReady((info) => {
   if (info.host === Office.HostType.Excel) {
-    document.getElementById("getGraphDataButton").onclick = ssoAuthHelper.getGraphData;
+    document.getElementById("getProfileButton").onclick = run;
   }
 });
+
+export async function run() {
+  getGraphData(writeDataToExcel);
+}
+
+function writeDataToExcel(result) {
```

Comment:
In typescript, this function is called `writeDataToOfficeDocument`. Let's make the naming consistent between both.